yorkie/backend/sync/etcd/locker.go

```diff
@@ -67,9 +67,8 @@ func (il *internalLocker) Unlock(ctx context.Context) error {
 		log.Logger.Error(err)
 		return err
 	}
-	if err := il.session.Close(); err != nil {
-		return err
-	}
 
-	return nil
+	var err = il.session.Close()
+
+	return err
```

Comment:
As suggested by `revive`, I don't think there's much of a problem with returning directly.

```suggestion
	return il.session.Close()
```